src/components/HeaderGradeDisplay.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface Props {
+	label: string;
+	grade: string;
+}
+const HeaderDisplayContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+`;
+const Label = styled.p`
+	font-size: ${({ theme }) => theme.fontSizes.xs};
+	color: ${({ theme }) => theme.colors.primary[0]};
+	width: 17em;
+	margin: 0;
+	margin-bottom: 0.5em;
+`;
+const HeaderGradeDisplayContainer = styled.div`
+	background-color: ${({ theme }) => theme.colors.main[0]};
+	width: 15em;
+`;
+const GradeDisplayText = styled.h2`
+	color: ${({ theme }) => theme.colors.gray[3]};
+	padding-left: 1em;
+`;
+function OpenInfo() {
+	console.log('opened!');
+}
+function HeaderGradeDisplay({ label, grade, ...props }: Props) {
+	return (
+		<HeaderDisplayContainer {...props}>
+			<Label>{label}</Label>
+			<HeaderGradeDisplayContainer onClick={OpenInfo}>
+				{grade == '' && <GradeDisplayText>N/A</GradeDisplayText>}
+				{grade != '' && <GradeDisplayText>{grade}%</GradeDisplayText>}
```

Comment:
Use `===` and `!==` here (strict equals, types must be the same). This is one of the crazy js things: ` '' == 0` evaluates to true!

I would also possibly try describing it this way:

`{ grade === '' && <Component1 /> || <Component2 /> }` although it might not like this;

If that doesn't work, you can also do a ternary operator:

` { grade === '' ? <Component1 /> : <Component2 /> }`

Sometimes with our formatter, it might not look as nice, which kind of sucks... but I think the ternary shows that only one or the other should render, rather than having two independent statements. And it stops people from messing with the logic and having two things render.